docs/rest-api/index.md

```diff
@@ -72,7 +72,39 @@ The company ID is a unique identifier that identifies the company. To find your
 3. Click on **Advanced Settings** on the left hand side
 4. Click on **API Settings**
 
+### Request Format
+
+The Bookingbug API is a JSON API. You must supply a `Content-Type: application/json` header in PUT and POST requests. You must set an `Accept: application/json` header on all requests.
+
+### Data Types
+
+The API returns and accepts JSON values, which can be strings in double quotes, numbers, objects, arrays, true or false. Most programming languages have tools to parse this data.
+
+### Time stamps
+
+Time stamps use UTC time and are formatted as ISO 8601 strings. Example: `2016-05-16T09:14:57Z`.
+
+### Pagination
+
+By default, most list endpoints return a maximum of 30 records per page. You can change the number of records on a per-request basis by passing a `per_page` parameter in the request URL parameters. Example: `per_page=50`.
+
+When the response exceeds the per-page maximum, you can paginate through the records by incrementing the page parameter. Example: `page=3`. List results include `next` and `previous` URLs in the response body for easier navigation:
+
+<pre>
+	  },
+  "_links": {
+	...
+    },
+    "next": {
+      "href": "https://uk.bookingbug.com/api/v1/admin/50666/bookings?page=3&per_page=30&locale=en"
+    },
+    "previous": {
+      "href": "https://uk.bookingbug.com/api/v1/admin/50666/bookings?page=1&per_page=30&locale=en"
+    }
+  }
+}
+</pre>
 
 This documentation will cover the main aspects of the API and how you can use it within your application. There are many different features within the API and we will only be covering some of the aspects that you will need to get set up. If you have further questions regarding the use of the API then you can email apisupport@bookingbug.com
 
-For a full list of end points and methods see the [API Reference](http://apidocs.bookingbug.com/)
```

Comment:
We shouldn't be changing the docs URL. If the new docs are ready, we should host them in the same place.